src/BalanceTx/BalanceTx.purs

```diff
@@ -103,21 +103,23 @@ import Wallet (cip30Wallet)
 -- | address.
 balanceTx
   :: UnattachedUnbalancedTx
+  -> UtxoMap
   -> QueryM (Either BalanceTxError FinalizedTransaction)
-balanceTx unbalancedTx = do
+balanceTx unbalancedTx utxos = do
```

Comment:
I suggest renaming `utxos` to `additionalUtxos` in `balanceTx`, `balanceTxWithAddress` and `runBalancer` functions.